src/main/services/lcp.ts

```diff
@@ -164,25 +172,40 @@ export class LcpManager {
         );
 
         // Update again lsd status
-        lsdStatus = await this.getLsdStatus(newPublicationDocument);
+        lsdStatus = await this.getLsdStatus(newPublicationDocument.data);
+        if (lsdStatus.isFailure) {
+            throw new Error(`Http getLsdStatus error with code
+                ${lsdStatus.statusCode} for ${lsdStatus.url}`);
+        }
         newPublicationDocument = await this.updateLsdStatus(
             publicationDocument,
-            lsdStatus,
+            lsdStatus.data,
```

Comment:
Due to the changed returned type of `updateLsdStatus()` and `getLsdStatus()`, the `.data` accessor should indeed be added to all instances. However in the current PR, there are missing ones. The program compiles fine because `lsdStatus` is `any`, so TypeScript does not see anything different between `lsdStatus` and `lsdStatus.data`. But the code will crash at runtime due to `lsdStatus` being a wrapper data structure instead of the actual `lsdStatus.data` object.